src/pages/mine.tsx

```diff
@@ -1,11 +1,54 @@
-import { FC } from 'react';
+import { FC, useState, useEffect } from 'react';
 import Head from '@/components/head';
 import Layout from '@/components/Layout';
+import Card from '@/components/tasks/mine_card';
```

Comment:
Seems like we don't need the mine_card component as described in previous comment
`import Card from '@/components/tasks/card';`